### ISSUE TITLE
MenuItem selection on iOS + VO "double presses" on first interaction

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#674]
From https://github.com/phetsims/scenery/issues/1094, @jessegreenberg and I saw that the most recent solution causes strange behavior when selecting a dialog to open for the first time. When double tapping to open the AboutDialog, the dialog opens and then immediately closes. Subsequent presses keeps the dialog open. When logging PDOM events, we were never getting a "click" event. This should block publication for anything trying to be published with iOS description support.

**Comment by zepumph** [comment]
@jessegreenberg should this block Ratio and Proportion?

**Comment by jessegreenberg** [comment]
Yes, I think so.

**Comment by zepumph** [comment]
I just spent about 15 minutes trying to reproduce on iOS 14.3 + VO and was not able to. I remember it being challenging when we first saw this, but I also remember that it was halfway through a change set over in https://github.com/phetsims/scenery/issues/1137. Perhaps it solved itself there? @jessegreenberg can you please try to reproduce, and recommend next steps.

**Comment by jessegreenberg** [comment]
I was able to reproduce with iOS 14.3 after pulling all changes. Ill take a look.

**Comment by jessegreenberg** [comment]
So for some reason, the first time I double the "About Dialog" menu item in the PhetMenu I get two down events - both at the same spot (presumably the center of the "About Dialog" menu item.  The second time I press the "About Dialog" menu item I only get one down event.

This is only happening for the "About Dialog" menu item, it is not happening for others like the "Check for Updates..." item.

**Comment by jessegreenberg** [comment]
Listing out notes from things I tried today:
- I tried to see this in another context, but haven't been able to duplicate. I tried both completely removing menu content from the DOM and also just using hidden. Here is some playground html
<details>

```html
<div class="menu_button" id="parent">
  <button id="openButton"
          aria-haspopup="true"
          aria-controls="menu2">
    WAI-ARIA Quick Links
  </button>
  <ul id="menu2"
      role="menu"
      aria-labelledby="menubutton">
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        W3C Home Page
      </button>
    </li>
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        W3C Web Accessibility Initiative
      </button>
    </li>
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        Accessible Rich Internet Application Specification
      </button>
    </li>
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        WAI-ARIA Authoring Practices
      </button>
    </li>
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        WAI-ARIA Implementation Guide
      </button>
    </li>
    <li role="none">
      <button class="menuButton" role="menuitem" href="">
        Accessible Name and Description
      </button>
    </li>
  </ul>
</div>

<div role="dialog" id="dialog">
  <button id="close">Close</button>
</div>

<script>
  const menu = document.getElementById( 'menu2' );
  const openButton = document.getElementById( 'openButton' );
  const closeButton = document.getElementById( 'close' );
  const parent = document.getElementById( "parent" );
  const dialog = document.getElementById( "dialog" );
  document.body.addEventListener( 'pointerdown', () => {
    console.log( 'down' );
  } );

  openButton.addEventListener( 'click', () => {
    parent.appendChild( menu );
  } );

  const menuButtons = document.getElementsByClassName( "menuButton" );
  for ( menuButton of menuButtons ) {
    menuButton.addEventListener( 'click', () => {
      parent.removeChild( menu );
      closeButton.focus();
    } );
  }

  // initially remove menu (after adding listeners)
  parent.removeChild( menu );

</script>
```
</details>

- I tried removing aria attibutes from menu items/the PhET menu but it had no impact.
- I tried an OptionsDialog as well, but it didn't happen for that Dialog either.
- I tried removing setting focus on the "close" button upon opening, no impact.
- I tried moving the "About..." menu item higher up in the PhET Menu list, no impact.

**Comment by jessegreenberg** [comment]
I just noticed that when I move the virtual cursor to the "About..." menu item VoiceOver plays a unique sound like two rising tones. When virtual cursor leaves the "About..." item it plays the same sound in reverse. So something is different about the "About..." menu item in particular.

**Comment by jessegreenberg** [comment]
I replaced the callback of the "About..." menu item with the callback for the "Check for Updates..." item and the problem went away. So it must be about the opening of the About dialog rather than menu items.

**Comment by jessegreenberg** [comment]
With this information, I tried all of these in the example HTML which did not make the problem happen:
- Copying the contents of the AboutDialog into the test.html example above.
- I tried adding/removing the dialog with links from the DOM.
- I tried creating a new link and adding it to the dialog every time it is opened.
- I tried creating a new Dialog with new HTML elements with links and inserting those into the DOM every time it is opened.

So something about having links in the AboutDialog is causing an extra down event to fire on the "About..." menu item. But I haven't isolated why it is happening yet.

**Comment by jessegreenberg** [comment]
Oh man...

This whole time my iPad was on iOS 14.2. Last night I updated to 14.3 and now the problem is gone. And that aligns with what @zepumph found when he tried to produce this. That should have been the first thing I verified. The bad news is that cost about 2 hours of pointless debugging. But the good news is there is nothing else for us to do here since it has been resolved in an Apple update.

Closing.
